model/docs/example_with_datasets.qmd

```diff
@@ -260,6 +259,63 @@ out = hosp_model.plot_posterior(
 )
 ```
 
+Test `posterior_predictive` and `prior_predictive` methods to generate posterior and prior predictive samples.
```

Comment:
We should also plot something from the prior predictive as an example.

Reply:
So, I plotted credible intervals for prior predictive distributions ujsing `plot_hdi`, and this is what it looks like:
![Capture](https://github.com/CDCgov/multisignal-epi-inference/assets/37636707/3ae07148-c432-49ab-8e81-d6fc978cace1)
Not a very compelling figure. For the purpose of demonstration, I am not aware of any arviz method to plot prior predictive distribution other than `plot_ppc`

Reply:
Log scale y will help here.

Reply:
Please also overlay the real data. Maybe it is already there but too tiny to see.

Reply:
I changed y scale and added observed data. I also removed the mean curve since the mean was outside hdi for some regions
![1](https://github.com/CDCgov/multisignal-epi-inference/assets/37636707/d87e45be-340a-4739-959c-b5ebd7a8480b)